docs/sections/user_guide/uw_yaml/forecast_yaml.rst

```diff
@@ -0,0 +1,232 @@
+.. _forecast_yaml:
+
+Forecast YAML
+=============
+
+The structured YAML to run a forecast is described below. It is enforced via JSON Schema.
+
+In this section, there are entries that define where the forecast will run (``run_directory:``), what will run (``executable:``), the data files that should be staged (``cycle-dependent:`` and ``static:``), and blocks that correspond to the configuration files required by the forecast model (``fd_ufs:``, ``field_table:``, ``namelist:``, etc.). Each of the configuration file blocks will allow the user to set a template file as input, or to define a configuration file in its native key/value pair format with an option to update the values (``update_values:``) contained in the original input file (``base_file:``).
+
+The configuration files required by the ufs-weather-model are documented :weather-model-io:`here<model-configuration-files>`.
```

Comment:
```suggestion
The configuration files required by the UFS Weather Model are documented :weather-model-io:`here<model-configuration-files>`.
```
Not sure if I'm right in suggesting this change -- the linked-to page has it as "UFS Weather Model", FWIW.

Reply:
Yep. That's good. I was stuck in "repo name" mode here.